catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/monitor/impl/WorkspaceQueryServiceImpl.java

```diff
@@ -150,6 +152,21 @@ public WorkspaceQueryServiceImpl(QueryUpdateSubscriber queryUpdateSubscriber,
         }
     }
 
+    public void setQueryTimeInterval(Integer queryTimeInterval) {
+        notNull(queryTimeInterval, "queryTimeInterval must be non-null");
+        if(queryTimeInterval < 0 || queryTimeInterval > 1440) {
+            LOGGER.debug("Invalid query time interval {}.  Defaulting to 1440 (24 hours).", queryTimeInterval);
+            queryTimeInterval = 1440;
```

Comment:
Instead of resettting the interval to 1440 after an invalid, I would say just keeping what it was before would be more consistent.